pkg/action/scan.go

```diff
@@ -33,17 +33,18 @@ func findFilesRecursively(ctx context.Context, root string, c Config) ([]string,
 				return nil
 			}
 			// False positives in refs file
-			if strings.Contains(path, "/.git/") {
+			if strings.HasPrefix(path, ".git/") {
```

Comment:
I believe this will inadvertently change the behavior, as I don't believe that "path" is relative here.

Reply:
Reverted in [`0a034a1` (#187)](https://github.com/chainguard-dev/bincapz/pull/187/commits/0a034a134f30c8641120899c9a8ce10ebcb374e3).